go/libraries/doltcore/env/repo_state.go

```diff
@@ -37,10 +37,10 @@ type RepoStateReader interface {
 }
 
 type RepoStateWriter interface {
-	// SetCWBHeadRef(context.Context, ref.DoltRef) error
 	// SetCWBHeadSpec(context.Context, *doltdb.CommitSpec) error
-	SetStagedHash(context.Context, hash.Hash) error
-	SetWorkingHash(context.Context, hash.Hash) error
+	SetStagedHash(hash.Hash) error
+	SetWorkingHash(hash.Hash) error
+	SetCWBHeadRef(ref.MarshalableRef) error
```

Comment:
Seems like we want to keep context for all these things... we're gonna end up adding it back in later otherwise.